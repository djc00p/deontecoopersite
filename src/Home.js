import React from "react";

const Home = () => {
  return (
    <div className="home-page page">
      <section>
        <h1>Welcome!</h1>
      </section>
    </div>
  );
};

export default Home;
