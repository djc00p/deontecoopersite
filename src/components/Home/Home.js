import React from "react";
import "./Home.css";

const Home = () => {
  return (
    <div className="home-page">
      <section>
        <h1>
          Welcome!
          <br />
          <br />
          Click a link above to learn more about me!
        </h1>

        <div className="image-container">
          <img
            src="https://imgur.com/G6pbL3R.jpg?1"
            alt="Head Shot"
            className="headshot"
          />
        </div>
      </section>
    </div>
  );
};

export default Home;
