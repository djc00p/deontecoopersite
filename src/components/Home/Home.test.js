import React from "react";
import { shallow, mount, render } from "enzyme";
import Home from "./Home";

describe("Home", () => {
  it("should create snapshot of Home", () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});
