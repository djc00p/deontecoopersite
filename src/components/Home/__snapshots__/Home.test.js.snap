// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should create snapshot of Home 1`] = `
<div
  className="home-page"
>
  <section>
    <h1>
      Welcome!
      <br />
      <br />
      Click a link above to learn more about me!
    </h1>
    <div
      className="image-container"
    >
      <img
        alt="Head Shot"
        className="headshot"
        src="https://imgur.com/G6pbL3R.jpg?1"
      />
    </div>
  </section>
</div>
`;
